src/Sitecore.AspNetCore.SDK.LayoutService.Client/Serialization/Fields/JsonSerializedField.cs

```diff
@@ -18,6 +18,12 @@ public class JsonSerializedField(JsonDocument doc)
 
     private readonly string _json = doc != null ? doc.RootElement.GetRawText() : throw new ArgumentNullException(nameof(doc));
 
+    /// <inheritdoc/>
```

Comment:
Can we do a clear doc instead of inheriting the default? "string representation" isn't really the real outcome here. Something along the lines of "Returns the raw json string which this field wraps"?

Reply:
Or perhaps we should consider having it as "GetRawValue()"? Is there a specific reason an override of "ToString()" is preferred?

Reply:
I've added docs and renamed it to `GetRawValue()`. I didn't have a specific reason for `ToString()`.